op-program/chainconfig/chaincfg.go

```diff
@@ -21,48 +22,48 @@ func OPSepoliaChainConfig() *params.ChainConfig {
 //go:embed configs/*json
 var customChainConfigFS embed.FS
 
-func RollupConfigByChainID(chainID uint64) (*rollup.Config, error) {
-	config, err := rollup.LoadOPStackRollupConfig(chainID)
+func RollupConfigByChainID(chainID eth.ChainID) (*rollup.Config, error) {
+	config, err := rollup.LoadOPStackRollupConfig(eth.EvilChainIDToUInt64(chainID))
```

Comment:
generally think `Unsafe` prefix is more idiomatic than `Evil`

Reply:
I'm going to be controversial and stick with `Evil` here for two reasons:
1. Every instance of `Unsafe` in our codebase refers to the unsafe head.
2. `Unsafe` implies that there are certain precondition checks you might do to ensure this is ok to call, but in this case it's never ok as it will always mean that 32 byte chain IDs are unsupported.